fn_query_tar_network/tests/test_query_tor_network.py

```diff
@@ -0,0 +1,49 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
```

Comment:
let's remove this file is it's not used.

Reply:
Raising New code Request hence closing the PR